pkg/configuration/configuration.go

```diff
@@ -225,8 +229,9 @@ func (c *Config) ClearBearer() error {
 	if err := keyring.DeleteBearer(h); err != nil {
 		return err
 	}
-	viper.Set("bearer", "")
-	viper.Set("expires_at", "")
```

Comment:
this was the issue, since it tries to reset config value in file, but seeing how we use it, we want the values that is already read into the struct to be null